packages/webpack-inline-constant-exports-plugin/index.js

```diff
@@ -148,9 +149,29 @@ class InlineConstantExportsPlugin {
 							 *
 							 * When inlining a constant, we remove the ImportSpecifierDependency and replace it
 							 * with a ConstDependency that inserts the inlined code verbatim.
+							 *
+							 * ---
+							 *
+							 * Webpack 5 keep tracks of re-exported dependencies:
+							 *   //const.js
+							 *   export const FOO=1;
+							 *
+							 *   //api.js
+							 *   export * as api from 'const';
+							 *
+							 *   //index.js
+							 *   import { api } from 'api';
+							 *   console.log( api.FOO );
+							 *
+							 * But this plugin does not support them. If a dependency has multiple IDs it means it is a
+							 * re-exported dep (eg: `api.FOO`), so we skip it.
 							 */
-							if ( dep instanceof HarmonyImportSpecifierDependency ) {
-								if ( dep.module.constantExports && dep.id in dep.module.constantExports ) {
+							if ( dep instanceof HarmonyImportSpecifierDependency && dep.ids.length === 1 ) {
```

Comment:
https://github.com/webpack/webpack/blob/4ba6d2ea67e9046cbba83f588989b3cadff7d911/lib/dependencies/HarmonyImportSpecifierDependency.js#L48

`id` is now `ids[]`